src/fluxo.object_store.js

```diff
@@ -90,74 +87,6 @@ class ObjectStore {
     }
   }
 
-  subscribe (eventName, callback) {
-    if (typeof callback !== "function") {
-      throw new Error("Callback must be a function");
-    }
-
-    this.events[eventName] = (this.events[eventName] || []);
-
-    this.events[eventName].push(callback);
-
-    return this.removeSubscription.bind(this, eventName, callback);
-  }
-
-  removeSubscription (eventName, callback) {
-    let index = this.events[eventName].indexOf(callback);
-
-    this.events[eventName].splice(index, 1);
-
-    if (!this.events[eventName]) {
-      delete this.events[eventName];
-    }
-  }
-
-  publish (eventName, ...args) {
-    var callbacks = this.events[eventName];
-
-    if (!callbacks) { return; }
-
-    for (let i = 0; i < callbacks.length; i++) {
-      callbacks[i].apply(null, args);
-    }
-  }
-
-  on (events, callback) {
-    var cancelers = [];
-
-    for (let i = 0, l = events.length; i < l; i++) {
-      let eventName = events[i],
-          changeEventToken = eventName,
-          canceler = this.subscribe(changeEventToken, callback.bind(this));
-
-      cancelers.push(canceler);
-    }
-
-    var aggregatedCanceler = function() {
-      for (let i = 0, l = cancelers.length; i < l; i++) {
-        let canceler = cancelers[i];
-        canceler.call();
-      }
-    };
-
-    this.signedEventsCancelers.push(aggregatedCanceler);
-
-    return aggregatedCanceler;
-  }
-
-  triggerEvents (eventsNames, ...args) {
-    for (let i = 0, l = eventsNames.length; i < l; i++) {
-      let eventName = eventsNames[i];
-      this.triggerEvent(eventName, ...args);
-    }
-  }
-
-  triggerEvent (eventName, ...args) {
-    this.publish(eventName, this, ...args);
-
-    this.publish("*", eventName, this, ...args);
-  }
-
```

Comment:
I removed these methods from the public interface as they are all available at `store.radio`.

As we talked before, maybe the event API should not even be public at all.

What are your final thoughts on that?

I think it's okay if we make the contract that `store.radio` is the interface to the events, as some people say, law of demeter should not be a law but a recommendation, there is some cases which makes sense to violate it (like this one).